### ISSUE TITLE
logger: messages are logged with priority kern.* by default (severe regression)

### ISSUE THREAD

**Issue by rgerhards** [karelzak/util-linux#163]
According to the man page, user.notice should be the default.

Reproduce:
$ logger message

will result in:
```
<13>Mar  5 16:01:38 rger: test 2.20
<5>Mar  5 16:01:59 logger: test 2.26
```
As can be seen, 2.26 logs with kernel prority (0 - for calculation see RFC3164/5424).

This is a severe regression, as log messages will shop up in the wrong files or show up nowhere. This may also confuse log analysis programs.

This is probably a regression from 1d57503378bdcd838365d625f6d2d0a09da9c29d where the syslog(3) calls were replaced with local code.

**Comment by rgerhards** [comment]
As I just see, the tag value is obviously also different from the past standard value.

**Comment by karelzak** [comment]
Thanks!   I'll add all the fixes to v2.26.1 and release it ASAP.

**Comment by rgerhards** [comment]
On the release: there are another couple of edges that look suspicious. I try to investigate this tomorrow. I would suggest to see what follows from that. But I think the most pressing problem-by-default regressions are now addressed.

**Comment by karelzak** [comment]
I have no any strict schedule for the release, so it's fine to wait and fix all known problems.

**Comment by karelzak** [comment]
Ad tag value, syslog(3) uses program name by default (that's probably reason for the change in commit 1d57503378bdcd838365d625f6d2d0a09da9c29d), but Sami probably overlooked that we have called openlog(getlogin(), ... ) before syslog(3).

Fixed by commit 17d66ccc7516a5e2cf681abc73406559da41eaae. Thanks for your report.
